src/plugins/checkboxes/checkbox.scss

```diff
@@ -0,0 +1,137 @@
+.provisional {
+	label {
+		/* (0,1,1) */
+		cursor: pointer;
+		display: block;
+		font-size: 20px;
+		line-height: 2;
+	}
+
+	fieldset.chkbxrdio-grp legend {
+		/* (0,2,2) */
+		float: none;
+		font-size: 22px;
+		font-weight: 400;
+		margin-bottom: 15px;
+		margin-top: 0;
+	}
+
+	[type="radio"],
+	[type="checkbox"] {
+		/* (0,2,0) */
+		border: 0;
+		clip: rect(0 0 0 0);
+		height: 1px;
+		margin: -1px;
+		overflow: hidden;
+		padding: 0;
+		position: absolute;
+		width: 1px;
+	}
+
+	[type="radio"] + span,
+	[type="checkbox"] + span,
+	[type="radio"] + label,
+	[type="checkbox"] + label {
+		/* (0,2,1) */
+		display: block;
+		margin-left: 36px;
+	}
+
+	[type="radio"] + {
+		span::before,
+		label::before {
+			/* (0,2,2) */
+			border: 4px solid #fff;
+			box-shadow: 0 0 0 2px #000;
+			content: "";
+			display: inline-block;
+			height: 36px;
+			left: 0;
+			position: absolute;
+			top: 0;
+			width: 36px;
+		}
+	}
+
+	[type="checkbox"] + {
+		span::before, label::before {
+			/* (0,2,2) */
+			border: 4px solid #fff;
+			box-shadow: 0 0 0 2px #000;
+			content: "";
+			display: inline-block;
+			height: 36px;
+			left: 0;
+			position: absolute;
+			top: 0;
+			width: 36px;
+		}
+	}
+
+	[type="radio"] {
+		+ {
+			span::before,
+			label::before {
+				/* (0,2,2) */
+				border-radius: 50%;
+			}
+		}
+
+		&:hover + span::before,
+		+ label:hover::before {
+			/* (0,3,1) */
+			background-image: linear-gradient(to bottom, hsl(10, 0%, 90%), hsl(10, 0%, 100%) 50%);
+		}
+	}
+
+	[type="checkbox"] {
+		&:hover + span::before,
+		+ label:hover::before {
+			/* (0,3,1) */
+			background-image: linear-gradient(to bottom, hsl(10, 0%, 90%), hsl(10, 0%, 100%) 50%);
+		}
+	}
+
+	[type="radio"]:focus + {
+		span::before,
+		label::before {
+			/* (0,3,2) */
+			box-shadow: 0 0 0 2px #000, 0 0 8px 4px rgb(59, 153, 252);
+		}
+	}
+
+	[type="checkbox"]:focus + {
+		span::before,
+		label::before {
+			/* (0,3,2) */
+			box-shadow: 0 0 0 2px #000, 0 0 8px 4px rgb(59, 153, 252);
+		}
+	}
+
+	[type="radio"]:checked + {
+		span::before,
+		label::before {
+			/* (0,3,2) */
+			background: #444;
+		}
+	}
+
+	[type="checkbox"]:checked + {
+		span::after,
```

Comment:
Can you try something like this to avoid using a span?

EXPLICIT
`[type="checkbox"]:checked + label::after`

IMPLICIT
`label [type="checkbox"]:checked::after`

Reply:
Unfortunately, only explicit label can be supported with that approach, because we cannot target a parent element of a checked element with `CSS`